modules/dxp/apps/portal-search-similar-results/portal-search-similar-results-web/src/main/resources/META-INF/resources/similar/results/configuration.jsp

```diff
@@ -77,23 +77,44 @@ SimilarResultsPortletPreferences similarResultsPortletPreferences = new SimilarR
+				<aui:input helpMessage="max-doc-freq-help" label="max-doc-freq" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_MAX_DOC_FREQUENCY) %>" type="number" value="<%= similarResultsPortletPreferences.getMaxDocFrequency() %>">
+					<aui:validator name="digits" />
+					<aui:validator name="min">1</aui:validator>
+				</aui:input>
 
-				<aui:input helpMessage="min-word-length-help" label="min-word-length" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_MIN_WORD_LENGTH) %>" type="text" value="<%= similarResultsPortletPreferences.getMinWordLength() %>" />
+				<aui:input helpMessage="min-word-length-help" label="min-word-length" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_MIN_WORD_LENGTH) %>" type="number" value="<%= similarResultsPortletPreferences.getMinWordLength() %>">
+					<aui:validator name="digits" />
+					<aui:validator name="min">0</aui:validator>
+				</aui:input>
 
-				<aui:input helpMessage="max-word-length-help" label="max-word-length" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_MAX_WORD_LENGTH) %>" type="text" value="<%= similarResultsPortletPreferences.getMaxWordLength() %>" />
+				<aui:input helpMessage="max-word-length-help" label="max-word-length" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_MAX_WORD_LENGTH) %>" type="number" value="<%= similarResultsPortletPreferences.getMaxWordLength() %>">
+					<aui:validator name="digits" />
+					<aui:validator name="min">1</aui:validator>
+				</aui:input>
 
 				<aui:input helpMessage="stop-words-help" label="stop-words" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_STOP_WORDS) %>" type="text" value="<%= similarResultsPortletPreferences.getStopWords() %>" />
 
 				<aui:input helpMessage="min-should-match-help" label="min-should-match" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_MIN_SHOULD_MATCH) %>" type="text" value="<%= similarResultsPortletPreferences.getMinShouldMatch() %>" />
 
-				<aui:input helpMessage="term-boost-help" label="term-boost" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_TERM_BOOST) %>" type="text" value="<%= similarResultsPortletPreferences.getTermBoost() %>" />
+				<aui:input helpMessage="term-boost-help" label="term-boost" name="<%= PortletPreferencesJspUtil.getInputName(similarResultsPortletPreferences.PREFERENCE_KEY_TERM_BOOST) %>" type="text" value="<%= similarResultsPortletPreferences.getTermBoost() %>">
+					<aui:validator name="number" />
+					<aui:validator name="min">[0]</aui:validator>
```

Comment:
Was there a reason brackets were added to this? I'm not seeing any difference between setting it to `0` or `[0]`.